benchmarks/dynamo/pr_time_benchmarks/benchmark_base.py

```diff
@@ -51,44 +53,68 @@
 
 
 class BenchmarkBase(ABC):
+    # measure total number of instruction spent in _work.
     _instruction_count = False
 
+    # measure total number of instruction spent in convert_frame.compile_inner
+    # TODO is there other parts we need to add ?
+    _compile_time_instruction_count = False
```

Comment:
This name is a bit ambiguous (is it a enable? Is it a value?) maybe something like `_enable_compile_time_instruction_count`? (Same issue w/ `_instruction_count` above)